cli/src/deploy/mod.rs

```diff
@@ -189,22 +196,24 @@ pub async fn info() {
     "deployConfig": deploy_configs,
   });
   let resp = post_v1("info", body).await;
-  match &resp {
-    Ok(res) => {
-      if res == REQUEST_TIMEOUT {
+  let resp = match &resp {
+    Ok(resp) => resp,
+    Err(err) => match err {
+      PostV1Error::Timeout => {
         println!("{}", REQUEST_TIMEOUT);
```

Comment:
should probably use `eprintln` to print to stderr instead of stdout

Reply:
Done!